scripts/health_check.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+#example usage: health_check.sh --network=main --min_height=52000 --version=2.1.0 18.136.37.63
+set -eo pipefail
+
+declare -A network_genesis_hashmap
+network_genesis_hashmap['main']="kh_pbtwgLrNu23k9PA6XCZnUbtsvEFeQGgavY4FS2do3QP8kcp2z"
+network_genesis_hashmap['uat']="kh_wUCideEB8aDtUaiHCtKcfywU6oHZW6gnyci8Mw6S1RSTCnCRu"
```

Comment:
maybe also hardcode min_height for those networks ?

Main at `54500` and uat at `54400`, @lucafavatella what's your opinion ?